db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/network_interface_plugged_into_vm.yaml

```diff
@@ -9,4 +9,4 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/event_action_refresh?target=ems"
+      value: "/System/event_handlers/event_action_refresh?target=src_vm"
```

Comment:
Okay that can probably be its own PR then since it is unrelated to these changes it will probably get merged faster